vcf-dumper/README.md

```diff
@@ -0,0 +1,122 @@
+</profile>
+```
+
+To build the project artifacts, execute `mvn package -P *chosen-profle*` in the vcf-dumper directory. An executable jar will be generated in *vcf-dumper-cli/target* and a war file in *vcf-dumper-ws/target*. The war file has been tested successfully in Apache Tomcat 9.
+
+## Test
+In order to test the VCF dumper, a MongoDB server containing variants in [EVA format](https://github.com/EBIvariation/eva-pipeline/wiki/MongoDB-schema) is needed. Some small test databases dumps are provided in the [test resouces](vcf-dumper-lib/src/test/resources/dump). Those dumps can be imported to a MongoDB server using the `mongorestore` command. **Those databases can be deleted if the tests are executed** (e.g., by `maven package`). There are several solutions for this: rename those databases, skip the tests when executing maven or execute *mavenrestore* after building the artifacts.
+
+Once we got a server with data, and a jar or war artifact pointing to it, we can try some queries.
+
+### Queries
+The VCF dumper has two main interfaces that can be used to execute queries over the archive and get the result in VCF format:
+* **WebServices**: allows to dump the variants of some study(ies) that are in a given genomic region (or list of regions). Queries with no region are not allowed
+* **Command Line Interface**: for EVA internal use only. Allows to dump all the variants for a given study(ies) and file(s). It does not include region filters
+
+#### Webservices
+The Webservices API contains actually a single endpoint: `{baseURL}/{regionId}/variants`.
+{regionId} can be a single region or a comma separated list of regions. Each region is a composed of a chromosome name, and optionally a ':' character followed by natural numbers pair (start and end), separated by '-'. Some examples of valid regions are:
+* `1`
+* `chr1`
+* `MT`
+* `1:1000-2000`
+* `chr1:1000-2000`
+
+And an example of a region list (whole chromosome 1, chromosome 2 positions 1000 to 2000, chromosome X positions 4000 to 5000) :
+* `1,2:1000-2000,X:4000-5000`
+
+The variants endpoint also include some URL variables. Two are mandatory:
+* **species**: the queried database name will be the species value prefixed by "eva_" (e.g., for species *hsapiens*, this service will fetch the data from a database named *eva_hsapiens*)
+* **studies**: comma separated list of studies ids to query
+
+In addition to those, there are several optional URL variables:
+* annot-ct
+* maf
+* polyphen
+* sift
+* ref
+* alt
+* miss_alleles
+* miss_gts
+
+Any time a valid call is invoked, the client will receive a VCF file stream, containing the variants that satisfy the query criteria. 
+#### CLI
+The Command Line Interface is intended to dump whole studies in VCF format. It does not allow filtering by region, so all the variants in the study will be dumped. This may be a time consuming operation, especially for big studies. For that reason, this tool is thought to be used only internally by EVA.
```

Comment:
"command line interface" should not be capitalized.

> this tool is thought to be used only internally by EVA.

What if somebody deploys their own instance of the EVA infrastructure? We shouldn't prevent them from using the CLI.